lampo-common/src/conf.rs

```diff
@@ -179,27 +177,32 @@ impl TryFrom<String> for LampoConf {
             .unwrap_or("nakamoto".to_owned());
         // Strip the value of whitespace
         let node = node.to_trimmed();
+        let network = Network::from_str(&network)?;
 
-        let mut core_url = None;
-        let mut core_user = None;
-        let mut core_pass = None;
-        if node == "core" {
-            core_url = conf
-                .get_conf("core-url")
-                .map_err(|err| anyhow::anyhow!("{err}"))?;
-            // If the value isn't none, strip the value of whitespace
-            core_url = core_url.map(|url| url.to_trimmed());
+        // Get the default conf
+        let mut bitcoind_conf = BitcoindConf::get_default_conf(network);
 
-            core_user = conf
-                .get_conf("core-user")
-                .map_err(|err| anyhow::anyhow!("{err}"))?;
-            core_user = core_user.map(|user| user.to_trimmed());
+        // Override if there is some config specified!
+        if let Some(bitcoin_url) = conf
+            .get_conf("core-url")
+            .map_err(|err| anyhow::anyhow!("{err}"))?
+        {
```

Comment:
split 
```suggestion
        let bitcoin_url = conf
            .get_conf("core-url")
            .map_err(|err| anyhow::anyhow!("{err}"))?;
            
        if let Some(bitcoin_url) = bitcoin_url {
```

is much readble in this way